docs/testplan/Smartswitch-test-plan.md

```diff
@@ -0,0 +1,821 @@
+DPU0       1     Online               dpu_midplane_link_state        up              Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_booted_state               up              Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_control_plane_state        up              Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_data_plane_state           up              Wed 20 Oct 2023 06:52:28 PM UTC    
+
+
+DPU1       2     Online               dpu_midplane_link_state        up              Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_booted_state               up              Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_control_plane_state        up              Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_data_plane_state           up            Wed 20 Oct 2023 06:52:28 PM UTC    
+
+root@sonic:~#show system-health DPU 0
+ 
+Name       ID    Oper-Status          State-Detail                   State-Value     Time                               Reason
+DPU0       1     Offline              dpu_midplane_link_state        down            Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_booted_state               down            Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_control_plane_state        down            Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_data_plane_state           down            Wed 20 Oct 2023 06:52:28 PM UTC    
+                                      
+root@sonic:~#show system-health DPU 0
+ 
+Name       ID    Oper-Status          State-Detail                   State-Value     Time                               Reason
+DPU0       1     Partial Online       dpu_midplane_link_state        up              Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_booted_state               up              Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_control_plane_state        up              Wed 20 Oct 2023 06:52:28 PM UTC
+                                      dpu_data_plane_state           down            Wed 20 Oct 2023 06:52:28 PM UTC    Pipeline failure
+
+
+```
+#### Pass/Fail Criteria
+ * Verify the following criteria for Pass/Fail:
+ * Online : All states are up
+ * Offline: dpu_midplane_link_state or dpu_booted_state is down
+ * Partial Online: dpu_midplane_link_state is up and dpu_booted_state is up and dpu_control_plane_state is up and dpu_data_plane_state is down
+ * Verify powering down DPU and check for status and powering up again to check the status to show online. 
+
+
+### 1.10 Check the Health of DPUs
+
+#### NOTE
+ * This Test case is to be covered in Phase 2
+
+#### Steps
+ *  Use command `show system-health detail <DPU_SLOT_NUMBER>` to check the health of the DPU.
+ 
+#### Verify in
+ * Switch
```

Comment:
@nissampa please add section for the expectation on the DPU host as well

Reply:
Discussed in the mail thread that we are focussing on NPU side first.